powerauth-webflow-authentication-mtoken/src/main/java/io/getlime/security/powerauth/lib/webflow/authentication/mtoken/exception/MobileTokenApiExceptionResolver.java

```diff
@@ -107,7 +107,7 @@
      * @return Response with error details.
      */
     @ExceptionHandler(SignatureVerificationFailedException.class)
-    @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
+    @ResponseStatus(HttpStatus.UNAUTHORIZED)
```

Comment:
Here, I don't understand why we have the new `SignatureVerificationException`. This is one of the few cases where we should use `PowerAuthAuthenticationException` - it should be thrown when signature validation fails or token auth validation fails...